### PR TITLE
chore: use cypress-multi-reporters

### PR DESCRIPTION
**Pull request by adeldhis2**
Integrate cypress-multi-reporters to enable ReportPortal and spec reporting for improved GitHub Actions logging

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,12 +30,14 @@
...
         "@testing-library/react": "^12.1.4",
         "cypress": "^13.10.0",
+        "cypress-multi-reporters": "^1.6.4",
         "cypress-tags": "^1.2.2",
         "enzyme": "^3.11.0",
         "enzyme-adapter-react-16": "^1.15.8",
         "eslint-plugin-cypress": "^2.12.1",
         "identity-obj-proxy": "^3.0.0",
         "jest-enzyme": "^7.1.2",
+        "mochawesome": "^7.1.3",
```
Comment (HendrikThePendric) [inline package.json:40]: Just out of interest, why did we need this additional requirement? Is it required by report portal?

**Review by HendrikThePendric — APPROVED** [review]
This is great @adeldhis2, thanks a lot. I left one question about an additional dependency, but since it's a devDependency it's not a critical point.

File: package.json (hunk 1 above)
Comment (adeldhis2) [reply package.json:40]: Good catch! I think this is not needed... let me remove it :)

**Comment by jenniferarnesen** [comment]
Report portal has been removed so this PR no longer makes sense.

**Pull request closed without merging**